ios/brave-ios/Sources/AIChat/Components/Markdown/Code Highlight/HighlightJS.swift

```diff
@@ -77,8 +77,8 @@ class HighlightJS {
         return processAttributedString(string, preferredFont: preferredFont)
       }
 
-      // Fallback to Apple's parser.
-      let styledCode = "<style>\(theme)</style><pre><code class='hljs'>\(code)</code></pre>"
+      // Fallback to Apple's parser. - Temporarily disabled due to crash: https://github.com/brave/brave-browser/issues/42443
+      /*let styledCode = "<style>\(theme)</style><pre><code class='hljs'>\(code)</code></pre>"
```

Comment:
If this is temporary should it have an issue to re-enable?

Reply:
It's a test PR just to disable to see if I can reproduce the crash. This crash ONLY happens on the CI builds, but never local or any other builds :(
So the idea is to disable it just for a single CI build. If it works, then I'll just completely remove this instead of temp.
If it doesn't work, then the crash is caused by something else in Brave-Core. It's almost impossible to get "proper" logs for it due to it only happening on TF builds which are optimized like crazy and symbolicating it gives no extra info.